opencti-platform/opencti-graphql/src/modules/case/case-incident/case-incident-resolvers.ts

```diff
@@ -0,0 +1,55 @@
+import type { Resolvers } from '../../../generated/graphql';
+import { buildRefRelationKey } from '../../../schema/general';
+import { RELATION_CREATED_BY, RELATION_OBJECT_ASSIGNEE, RELATION_OBJECT_LABEL, RELATION_OBJECT_MARKING } from '../../../schema/stixMetaRelationship';
+import { stixDomainObjectAddRelation, stixDomainObjectCleanContext, stixDomainObjectDelete, stixDomainObjectDeleteRelation, stixDomainObjectEditContext, stixDomainObjectEditField } from '../../../domain/stixDomainObject';
+import {
+  addCaseIncident,
+  caseIncidentContainsStixObjectOrStixRelationship,
+  findAll,
+  findById
+} from './case-incident-domain';
+
+const caseIncidentResolvers: Resolvers = {
+  Query: {
+    caseIncident: (_, { id }, context) => findById(context, context.user, id),
+    caseIncidents: (_, args, context) => findAll(context, context.user, args),
+    caseIncidentContainsStixObjectOrStixRelationship: (_, args, context) => {
+      return caseIncidentContainsStixObjectOrStixRelationship(context, context.user, args.id, args.stixObjectOrStixRelationshipId);
```

Comment:
I don't find where this method is used

Reply:
It was a query in the client python so we've decided to add it to the api too